contents/docs/user-guides/groups.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Groups
```

Comment:
Note: This isn't linked from any sidebar. Intentional?

Reply:
Fixed